.github/mergify.yml

```diff
@@ -1,16 +1,40 @@
----
 defaults:
   actions:
     queue:
       name: default
-      method: rebase
-      update_method: rebase
-
 queue_rules:
+  - name: duplicated default from design changes needs approval from at least one
+      core maintainer
+    queue_conditions:
+      - base~=^(main)|(release-.+)$
+      - label!=design
+      - "#approved-reviews-by>=1"
+      - "#changes-requested-reviews-by=0"
+      - check-success=test-lint
+      - check-success=test-protoc
+      - base~=^(main)|(release-.+)$
+      - label=design
```

Comment:
this `queue_conditions` does not look right. It has both
- `label!=design`
- `label=design`

`base` is 2x in the list as well as others.